buildSrc/src/main/java/org/elasticsearch/gradle/precommit/ThirdPartyAuditPrecommitPlugin.java

```diff
@@ -24,17 +24,30 @@
 import org.elasticsearch.gradle.info.BuildParams;
 import org.gradle.api.Project;
 import org.gradle.api.Task;
+import org.gradle.api.artifacts.Configuration;
 import org.gradle.api.tasks.TaskProvider;
 
 import java.nio.file.Path;
 
 public class ThirdPartyAuditPrecommitPlugin extends PrecommitPlugin {
+
+    public static final String JDK_JAR_HELL_CONFIG_NAME = "jdkJarHell";
+    public static final String LIBS_ELASTICSEARCH_CORE_PROJECT_PATH = ":libs:elasticsearch-core";
+
     @Override
     public TaskProvider<? extends Task> createTask(Project project) {
         project.getPlugins().apply(CompileOnlyResolvePlugin.class);
         project.getConfigurations().create("forbiddenApisCliJar");
         project.getDependencies().add("forbiddenApisCliJar", "de.thetaphi:forbiddenapis:2.7");
 
+        Configuration jdkJarHellConfig = project.getConfigurations().create(JDK_JAR_HELL_CONFIG_NAME);
+        if (BuildParams.isInternal() && !project.getPath().equals(":libs:elasticsearch-core")) {
```

Comment:
nit: We try to use `== false` instead of `!` as the former is easier to visually see when skimming through code.

Reply:
👍